<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="history_output">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>390</y>
      <width>451</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>控制栏</string>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>生成</string>
     </property>
    </widget>
    <widget class="QPushButton" name="input_picture">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>输入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SetMaterial">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>设置材料</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_parameter">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>设置参数</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>369</y>
      <width>481</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>日志</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>571</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>history_output</zorder>
   <zorder>progressBar</zorder>
   <zorder>qvtkWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>PushedButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>727</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>PushedButton()</slot>
 </slots>
</ui>
